src/homework2/task7.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
Missed tasks descriptions - failed sanity checks. 
We have to put tasks descriptions to file